.circleci/config.yml

```diff
@@ -28,37 +40,28 @@ commands:
           command: gpg --batch --import $GPG_KEY
       - run:
           name: Fixing GPG compatibility
-          command: echo $GPG_SIGNING_KEY_PW_NEW | gpg --batch --pinentry-mode=loopback --passphrase-fd 0 --export-secret-keys > /home/circleci/.gnupg/secring.gpg
+          command: |
+            echo $GPG_SIGNING_KEY_PW_NEW | gpg --batch --pinentry-mode=loopback --passphrase-fd 0 \
+            --export-secret-keys > /home/circleci/.gnupg/secring.gpg
 
 jobs:
-  build:
-    executor: android-executor
-    steps:
-      - checkout
-      - android/accept-licenses
+
   test:
     executor: android-executor
     steps:
       - checkout
-      - restore_cache:
-          key: jars-{{ checksum "build.gradle.kts" }}-{{ checksum  "purchases/build.gradle" }}-{{ checksum "common/build.gradle" }}-{{ checksum "public/build.gradle" }}-{{ checksum "utils/build.gradle" }}-{{ checksum "strings/build.gradle" }}
-      - run:
-          name: Download Dependencies
-          command: ./gradlew androidDependencies --no-daemon --max-workers=2
-      - save_cache:
-          paths:
-            - ~/.gradle
-          key: jars-{{ checksum "build.gradle.kts" }}-{{ checksum  "purchases/build.gradle" }}-{{ checksum "common/build.gradle" }}-{{ checksum "public/build.gradle" }}-{{ checksum "utils/build.gradle" }}-{{ checksum "strings/build.gradle" }}
+      - android/accept-licenses
+      - android-dependencies
       - android/restore-build-cache
       - run:
           name: Run Tests
-          command: ./gradlew lint test --no-daemon --max-workers=2
+          command: ./gradlew lint test
 #      - run:
 #          name: Test coverage report
 #          command: ./gradlew jacocoFullReport coveralls --no-daemon --max-workers=2
```

Comment:
any reason to keep this around? if so, maybe we should update the `no daemon` and `max workers` options

Reply:
I will remove it and come back to it in the future if we bring jacoco back